Lib/fontTools/feaLib/ast.py

```diff
@@ -591,6 +591,23 @@ def asFea(self, indent=""):
         return "ignore sub " + ", ".join(contexts) + ";"
 
 
+class IncludeStatement(Statement):
+    def __init__(self, location, filename):
+        super(IncludeStatement, self).__init__(location)
+        self.filename = filename
+
+    def build(self):
+        # REVIEW: check that raising makes sense here
+        # REVIEW: is this the correct exception class?
+        raise FeatureLibError(
```

Comment:
> is this the correct exception class?

it's the only one defined in `fontTools.feaLib.error`

Reply:
> It does not make sense to build an include statement.
> check that raising makes sense here

I wonder if instead of raising here, we could instead lazily load the including lexer and resolve the include statements when asked to "build" them? not sure if that's possible, just a thought

Reply:
> not sure if that's possible, just a thought

It sounds a bit complex as a solution given that I'm not sure anyone needs that use-case. I think if one wants to get include statements in the AST, they're not interested in building the features, and inversely if someone wants to build the features, there is already a solution for that.

I can rephrase the exception message and/or add a TODO in the code, saying "consider lazy-loading a parser in the future"?

Just FYI, the two use-cases we have in mind that require to not follow include statements are:
* slicing up the feature file into GSFeaturePrefixes and GSFeatures for glyphsLib
* normalizing feature files

Reply:
I think the commit message needs a bit more explanation about the use case this is trying to support, right now I don’t even know what is being addressed here.

Reply:
Done. Is it understandable?

Reply:
Clearer now, thanks!